src/locales/da.json

```diff
@@ -1,933 +1,932 @@
-        "Transparency": "Transparens",
-        "VoxelMode": "Voxel Mode (ASMBL)",
-        "SpecularLighting": "Spejlende belysning",
-        "ReloadRequired": "Opdatering påkrævet",
-        "LoadCurrentFile": "Aktuel fil",
-        "Tracking": "Sporing",
-        "Rendering": "Rendering",
-        "Downloading": "Downloader",
-        "ColorMode": "Farvevisning"
-    }
+{
+    "title": "Dansk",
+    "App": {
+        "Printers": "Printere",
+        "Titles": {
+            "Error": "Fejl",
+            "Pause": "Pauset",
+            "Printing": "{percent}% printet",
+            "PrintingETA": "{percent}% printet - ETA {eta}",
+            "Complete": "Færdig - {filename}"
+        },
+        "TopBar": {
+            "SAVE_CONFIG": "GEM KONFIGURATION",
+            "UploadPrint": "Upload & Print",
+            "EmergencyStop": "Nødstop",
+            "Uploading": "Uploader",
+            "UploadOfFileSuccessful": "Upload af {file} udført",
+            "CannotUploadTheFile": "Kan ikke uploade filen!"
+        },
+        "TopCornerMenu": {
+            "KlipperControl": "Klipper styring",
+            "KlipperRestart": "Genstart",
+            "KlipperFirmwareRestart": "Genstart firmware",
+            "ServiceControl": "Services",
+            "PowerDevices": "Strømstyring",
+            "HostControl": "Værtsstyring",
+            "Start": "Start",
+            "Stop": "Stop",
+            "Restart": "Genstart",
+            "Reboot": "Genstart",
+            "Shutdown": "Luk ned",
+            "Cancel": "Annuller",
+            "ConfirmationDialog": {
+              "Title": {
+                "KlipperRestart": "Genstart Klipper",
+                "KlipperFirmwareRestart": "Genstart Klipper firmware",
+                "ServiceStart": "Start service",
+                "ServiceRestart": "Genstart service",
+                "ServiceStop": "Stop service",
+                "HostReboot": "Genstart vært",
+                "HostShutdown": "Luk vært ned"
+              },
+              "Description": {
+                "KlipperStop": "Hvis du stopper Klipper nu, vil det igangværende print fejle.",
+                "KlipperRestart": "Hvis du genstarter Klipper nu, vil det igangværende print fejle.",
+                "KlipperFirmwareRestart": "Hvis du genstarter Klipper firmware nu, vil det igangværende print fejle.",
+                "ServiceStart": "Hvis du starter denne service nu, er der stor risiko for at dit igangværende print vil fejle.",
+                "ServiceRestart": "Hvis du genstarter denne service nu, er der stor risiko for at dit igangværende print vil fejle.",
+                "ServiceStop": "Hvis du stopper denne service nu, er der stor risiko for at dit igangværende print vil fejle.",
+                "HostReboot": "Hvis du genstarter værten nu, vil det igangværende print fejle.",
+                "HostShutdown": "Hvis du lukker værten ned nu, vil det igangværende print fejle."

```

Comment:
Can you please fix the indentation from line 34 to 51?
One indentation level is equal to 4 spaces. Indentations we see here are only 2 spaces.
Thank you in advance!